src/site/content/en/blog/window-controls-overlay/index.md

```diff
@@ -317,6 +317,25 @@ for example, `"standalone"`.
 
 {% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/1VonSc0jOiOureeSkqtf.png", alt="PWA running in standalone mode with the window controls overlay displayed in the body.", width="800", height="428" %}
 
+## UI considerations
+
+While it may be tempting, it is not recommended to try to create classic dropdown
+menu bars in the Window Controls Overlay area. Doing so would violate the
```

Comment:
```suggestion
While it may be tempting, creating a classic dropdown menu in the Window Controls Overlay area is not recommended. Doing so would violate the
```

Reply:
This is Strunk and White rule II-22. Place the emphatic words of a sentence at the end. This sounds counterintuitive. It allows the context of a point to sink in before the point lands.

Reply:
Wow, thanks!